website/docs/Getting-Started.md

```diff
@@ -19,7 +19,7 @@ AutoGen is powered by collaborative [research studies](/docs/Research) from Micr
 ### Quickstart
 
 Install from pip: `pip install pyautogen`. Find more options in [Installation](/docs/Installation).
-For [code execution](https://microsoft.github.io/autogen/FAQ#code-execution), we strongly recommend installing the python docker package, and using docker.
+For [code execution](/docs/FAQ#code-execution), we strongly recommend installing the python docker package, and using docker.
```

Comment:
Which style do we prefer? A URL (https://microsoft.github.io/autogen/docs/FAQ#code-execution) or a relative path? @sonichi @qingyun-wu

Reply:
Relative path is better. Thank you Li for asking!